qemu/tests/qemu_guest_agent_snapshot.py

```diff
@@ -8,6 +8,7 @@
 
 from qemu.tests.live_snapshot_basic import LiveSnapshot
 from qemu.tests.qemu_guest_agent import QemuGuestAgentBasicCheckWin
+from provider.blockdev_snapshot_base import BlockDevSnapshotTest
```

Comment:
move this import under line7, also add a new line  before and after this import 

import orders:
standard lib
avocado
virttest
provider

Never import a case level module, just like this tc does, e.g. from qemu.tests.live_snapshot_basic import LiveSnapshot